src/lib.rs

```diff
@@ -118,6 +121,10 @@ impl CrashReporter {
         let mut crash_list_json: serde_json::Value = json!(report.crash_list);
         let data = crash_list_json.as_object_mut().unwrap();
 
+        if self.filter_out_errors {
+            self.filter_out_errors(data);
```

Comment:
The missing _mut_ in line 122 means that the variable itself can't be re-assigned. If you add the _mut_, the compiler gives a warning, indicating that the _mut_ can be removed since data is not re-assigned.
The function `as_object_mut()`, return a hash map that is mutable in the sense that it can be manipulated, which is done in the following functions, like `filter_out_errors`. If each of those functions would return a new hashmap that would get re-assigned to `data`, the _mut_ would be required.